src/main/java/homework3/ex1/page/Ex1HomePage.java

```diff
@@ -0,0 +1,111 @@
+package homework3.ex1.page;
+
+import homework3.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import java.util.List;
+
+public class Ex1HomePage extends BasePage {
+
+    private WebDriver driver;
+
+    @FindBy(id = "user-icon")
+    private WebElement userIcon;
+    @FindBy(id = "name")
+    private WebElement loginField;
+    @FindBy(id = "password")
+    private WebElement passwordField;
+    @FindBy(id = "login-button")
+    private WebElement enterBtn;
+    @FindBy(id = "user-name")
+    private WebElement userName;
+
+
+    @FindBy(className = "benefit-icon")
+    private List<WebElement> benefitIcons;
+    @FindBy(className = "benefit")
+    private List<WebElement> benefitText;
+    @FindBy(xpath = "//div[@class = 'benefit']/span[contains(text(),'To include good practices')]")
```

Comment:
can be simplified:
//div[@class = 'benefit']/span[contains(text(),'To include good practices')] -> 
//span[contains(text(),'To include good practices')]